data.sql

```diff
@@ -36,3 +36,40 @@ INSERT INTO species(name) VALUES('Digimon');
  UPDATE animals SET owner_id=(SELECT id FROM owners WHERE full_name='Melody Pond') WHERE name IN('Charmander','Squirtle','Blossom');
  UPDATE animals SET owner_id=(SELECT id FROM owners WHERE full_name='Dean Winchester') WHERE name IN('Angemon','Boarmon');
 
+-- Vets data
+
+INSERT INTO vets(name,age,date_of_graduation) VALUES('William Tatcher',45,'2020-04-23');
```

Comment:
- [ ] Please make sure to insert the proper data given in the requirement as the vet "William" graduated on "2000-04-23" not in "2020".

Reply:
Fixed.